www/config.toml

```diff
@@ -2,6 +2,9 @@ baseURL = ""
 languageCode = "en-us"
 title = "MIT OpenCourseWare"
 
+# Make sure errors with getJSON don't stop the build
+ignoreErrors = ["error-remote-getjson"]
```

Comment:
It looks like the intention is to only do this for development: https://github.com/mitodl/ocw-hugo-themes/pull/1/files#r621530790

Is there a way to configure this option only in development environments?

Reply:
@Ferdi mentioned that the theme cannot depend on `ocw-studio` being up and running.  I do see your point though, that if there were an intermittent outage of `ocw-studio` and a full site build happened during that then there would be no new courses or news.  I think we can pull this off with environment based configuration as described [here](https://gohugo.io/getting-started/configuration/#configuration-directory) but then the production build would still completely stop if `ocw-studio` was unavailable.  Do we want that?

Reply:
> There's a race condition which happens when `npm start` is run, where hugo and webpack are started at the same time but hugo is looking for webpack files to create links for the javascript files. These files exist the second time hugo is started but they refer to the files created the first time webpack was started. I'm not sure it's worth holding up the PR for so I'll file an issue

Yea, I've experienced this and wasn't able to find a quick and easy solution but I think it's worth looking into outside of the scope of this PR.

Reply:
Also @noisecapella to answer your original question, config files are actually not needed in the theme at all, although I put some examples under `example_configs`.  The config should live in the project utilizing the theme.

Reply:
I don't think we ever want the production site to look like this:
![Screenshot from 2021-04-28 10-18-49](https://user-images.githubusercontent.com/863262/116450205-040cb800-a829-11eb-8590-44ae2666c8aa.png)

A build which failed due to an API outage from ocw-studio can be restarted, and in the meantime the site is functioning as normal. We cannot just decide not to rely on ocw-studio because we need the OCW news feed in order to create content for the cards on the frontpage, and the news items updates on a regular basis. We also rely on ocw-studio for course information for the carousel cards. The choice is to break the build or to display an error message on the production site. @Ferdi @pdpinch any thoughts?